src/clj/rems/db/workflow.clj

```diff
@@ -18,9 +17,8 @@
       (update-in [:workflow :handlers] #(mapv users/get-user %))))
 
 (defn get-workflow [id]
-  (-> {:wfid id}
-      db/get-workflow
-      enrich-and-format-workflow))
+  (when-let [wf (db/get-workflow {:wfid id})]
+    (enrich-and-format-workflow wf)))
```

Comment:
can the wf really be nil?

Reply:
Yes, if the ID is not found.

Reply:
There was a bug that the API returned error 500 instead of 404 when the workflow is not found.